### DISCUSSION TITLE
[QA] InterLM的依赖的版本需求

### DISCUSSION THREAD

**Discussion by DirtyKnightForVi** [InternLM/InternLM#660] [Q&A]
### Describe the question.

1. 4-bits版本的模型，必须要安装lmdeploy才能运行么？
2. 非量化版的transformers和tokenizers的版本要求是什么？你们文档中有些地方的版本是矛盾的。

**Comment by hellock** [comment]
#628

**Comment by ZwwWayne** [comment]
推荐使用 transformers==4.34

**Comment by ZwwWayne** [comment]
如果是使用 [InternLM2-Chat-20B-TurboMind-4Bits](https://huggingface.co/spaces/internlm/InternLM2-Chat-20B-TurboMind-4Bits)，需要使用 lmdeploy。其他情况下，可以参考 https://github.com/InternLM/InternLM?tab=readme-ov-file#import-from-transformers 借助 bitsandbytes
